### PR TITLE
HW 3. JavaScript primitive data types

### PR DESCRIPTION
**Pull request by viaylya1**

### CODE REVIEW

File: jsBasics/task1.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+// String
+const firstName = "Vika";
+let lastName = "Bryzhak";
+console.log(firstName, typeof firstName);
+console.log(lastName, typeof lastName);
+
+// Boolean
+let isMarried = false;
+let hasCar = true;
```
Comment (sttaran) [inline jsBasics/task1.js]: use `const`

File: jsBasics/task1.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+// Number
+const integerNumber = 5;
+const fractionalNumber = 5.5;
+const PI = 3.14;
+console.log(integerNumber, fractionalNumber, PI);
+console.log(typeof integerNumber, typeof fractionalNumber, typeof PI);
+
+// String
+const firstName = "Vika";
+let lastName = "Bryzhak";
```
Comment (sttaran) [inline jsBasics/task1.js]: use `const`

File: jsBasics/task3.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Numbers and Booleans
+
+let personAge = 14;
```
Comment (sttaran) [inline jsBasics/task3.js]: use `const`

File: jsBasics/task4.1.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// Area of Circle
+
+let radius = 10;
+let areaOfCircle = (Math.PI * Math.pow(radius, 2)).toFixed(2);
```
Comment (sttaran) [inline jsBasics/task4.1.js]: use `const`

**Review by sttaran — CHANGES_REQUESTED** [review]

File: jsBasics/task1.js (hunk 1 above)
Comment (viaylya1) [reply jsBasics/task1.js]: fixed

File: jsBasics/task1.js (hunk 2 above)
Comment (viaylya1) [reply jsBasics/task1.js]: fixed

File: jsBasics/task3.js (hunk 3 above)
Comment (viaylya1) [reply jsBasics/task3.js]: fixed

File: jsBasics/task4.1.js (hunk 4 above)
Comment (viaylya1) [reply jsBasics/task4.1.js]: fixed

**Review by sttaran — APPROVED** [review]

**Pull request merged**